e2e/cypress/integration/integrations/builtin_commands/common_commands_spec.js

```diff
@@ -288,37 +288,6 @@ describe('I18456 Built-in slash commands: common', () => {
         cy.uiWaitUntilMessagePostedIncludes(`You were added to the channel by @${user1.username}`);
     });
 
-    it('MM-T661 /invite extra white space before @ in DM or GM', () => {
```

Comment:
Was this accidentally removed?

Reply:
Oops. You're correct. I may be switching branches and press some buttons...Will fix!